lib/govuk_publishing_components/presenters/button_helper.rb

```diff
@@ -36,6 +38,8 @@ def html_options
         options[:data] = data_attributes if data_attributes
         options[:title] = title if title
         options[:target] = target if target
+        options[:name] = name if name.present? && value.present?
+        options[:value] = value if value.present? && name.present?
```

Comment:
Nitpick: I'd maybe switch the logic on this line around to match the previous one, just to increase the readability a little, i.e.

```
options[:name] = name if name.present? && value.present?
options[:value] = value if name.present? && value.present?
```

Reply:
Good idea - done.